README.md

```diff
@@ -83,7 +83,29 @@ $ nodemon bin/www
 ```shell
   http://localhost:3000/
 ```
+### Docker setup
 
+> Docker containerization 
```

Comment:
Idk what's happening with our Tech Stack section, but the way to handle sub-headers is just add another `#`

The `>` is used for quoting things (like lines from documentation links so people don't have to click into them)

And there are two ways to handle code styling.
1. In-line `looks like this` which involves wrapping in a single set of backticks ` `` `
2. Multi-line (I just use this if the code is on it's own line instead if in other text). Which involves 3 backticks ` ``` ... ``` `

You probably want these multi-line, and you can even hint at the type of the code going in. Since you're just running this in a terminal / shell. It's a good practice to just use `shell`.

All together. My suggestion for these sections would be.

#### Docker Containerization
```shell
docker build -t my-app --platform linux/amd64
```

Which in code is
```markdown
#### Docker Containerization
```shell
docker build -t my-app --platform linux/amd64
`` `
```
(Remove the extra space in the triple backtick)

## References
https://www.rubycoloredglasses.com/2013/04/languages-supported-by-github-flavored-markdown/
> shell, sh, zsh, bash - Shell scripting

Reply:
Idk what's happening with our Tech Stack section, but the proper way to write the documentation for any project is in the wiki's section of [github wiki](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis)

I have just appended the commands in the same way the old README was implemented.
For now we can keep it in the same way.
All together. My suggestion for these sections would be.
1. To create a wiki page.
2. Add proper multiple documentation about the repo and project.

Reply:
> but the proper way to write the documentation for any project is in the wiki's section of github wiki

Wikis and READMEs are different, you're currently updating a README, so my advice / request was about using the README correctly.

> I have just appended the commands in the same way the old README was implemented.

This is fine. I agree with this argument as we don't *need* to refactor the README in this PR. This can be handled in #51